src/Band/StringInstruments/Guitar.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace Band\StringInstruments;
+
+/**
+ * Class Guitar
+ */
+class Guitar extends StringInstrument
+{
+    function getNumberOfStrings()
+    {
+        return 6;
```

Comment:
Class Guitar is not abstract. It just extends abstract class StringInstrument. Abstract method getNumberOfStrings() is implemented in Guitar class. This method simply returns the number of strings (not chords) in the instrument.